packages/gallery/tests/e2e/styleParams/arrowsVerticalPosition.e2e.spec.js

```diff
@@ -0,0 +1,102 @@
+import GalleryDriver from '../../drivers/pptrDriver';
+import { toMatchImageSnapshot } from '../../drivers/matchers';
+import { GALLERY_CONSTS } from 'pro-gallery-lib';
+
+expect.extend({ toMatchImageSnapshot });
+
+// Relevant to entire test suite (both 'describe's)
+let driver;
+
+// gets relevant layout style params
+const getLayoutStylesByVerticalPosition = (
+  titlePlacement,
+  arrowsVerticalPosition
+) => {
+  return {
+    galleryLayout: GALLERY_CONSTS.layout.SLIDER,
+    titlePlacement: titlePlacement,
+    arrowsVerticalPosition: arrowsVerticalPosition,
+  };
+};
+
+describe('arrowsPosition with info below - e2e', () => {
+  beforeAll(async () => {
+    driver = new GalleryDriver();
+    await driver.openPage();
+  });
+
+  afterEach(async () => {
+    await driver.waitFor.hookToBeVisible('item-container');
+    const page = await driver.grab.elemScreenshot('.pro-gallery');
+    expect(page).toMatchImageSnapshot();
+  });
+
+  afterAll(async () => {
+    await driver.closePage();
+  });
+
+  it('should have arrows on info center', async () => {
+    const currentTestLayout = getLayoutStylesByVerticalPosition(
+      'SHOW_BELOW',
+      'INFO_CENTER'
+    );
+    await driver.navigate(currentTestLayout);
+  });
+
+  it('should have arrows on item center', async () => {
+    const currentTestLayout = getLayoutStylesByVerticalPosition(
+      'SHOW_BELOW',
+      'ITEM_CENTER'
+    );
+    await driver.navigate(currentTestLayout);
+  });
+
+  it('should have arrows on image center', async () => {
+    const currentTestLayout = getLayoutStylesByVerticalPosition(
+      'SHOW_BELOW',
+      'IMAGE_CENTER'
+    );
+    await driver.navigate(currentTestLayout);
```

Comment:
@yoavva are you sure something is being tested here?

Reply:
My bad! they weren't sensitive enough, they are now
@nirnaor

Reply:
1. What do you mean they are now? 
2. What do you mean sensitve?
3.  There's no expect that I'm seeing in this test?
It doesn't test anything @yoavva

Reply:
3. The expect is in the afterEach methods, I've put it there to prevent code duplication. 
1-2. The tests were not sensitive enough earlier which means that even if the arrows were not located where they should have, this change was small enough so that the tests will pass anyway (toMatchImageSnapshot was configured with 5% difference threshold so the arrows dislocated made less than 5% difference between the images due to their small size I guess) , however, I've added 'testsSensitivityConfig' to toMatchImageSnapshot(testsSensitivityConfig), which makes the tests not tolerable to any change and now they will fail the test for every minor dislocation of the nav arrows.
@nirnaor

Reply:
@yoavva 
`afterEach` is used to clean **after the test is finished**. You should not use it for assertions. 
I'm not even sure If tests will be marked as failures if their teardown fails. 
You need to `expect` inside the test. Just use a function that does that, but not `afterEach`.
P.S
I'm not a fan of `beforeEach` and `afterEach`. I think it makes the reading experience of tests terrible. I just prefer to reuse functions and to read each test from start to bottom.

Reply:
More about [it here](https://jestjs.io/docs/setup-teardown)

Reply:
Thanks for the explanation! 
Got it, no more testing in those functions. 
Changed it, can you take a look?
@nirnaor